templates/_layouts/_root.html

```diff
@@ -14,6 +14,7 @@
 
     <title>{% block title %}{{ title }}{% endblock %}{% if self.title() %} | {% endif %}Vanilla documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script src="https://assets.ubuntu.com/v1/703e23c9-lazysizes+noscript+native-loading.5.1.2.min.js" defer></script>
```

Comment:
Currently we have some lazy loading script in `templates/_layouts/site.html`, so I think we should keep it there (and remove the old one that uses Blazy).